docs/developing.rst

```diff
@@ -119,8 +123,79 @@ Start by following the steps here: https://github.com/edx/edx-proctoring
         * Go to ``/admin/verify_student/manualverification/`` on your sandbox
         * Create a record for the given user, with status "approved".
 
+Backend-specific Information
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+One of the main motivations for setting up a sandbox to test
+proctoring is having an externally accessible system which can be
+accessed by our proctoring providers' systems. This enables more
+thorough end-to-end testing.
+
+To enable proctoring in a way that won't be overridden by ansible
```

Comment:
After one sshing to the running sandbox and updating the yaml file, is any action required after that change?  Do you need to rerun ansible on the sandbox (e.g. as describe in the `Updating code` section of [this](https://openedx.atlassian.net/wiki/spaces/EdxOps/pages/13960183/Sandboxes) document)? Or does that occur as part of restarting the lms and cms processes?

Reply:
@MichaelRoytman do you think the update I put for this is enough?

Reply:
I think so! The only thing I notice is that the bullet points under `In addition to adding these configurations, you'll also need to set up
a user which PT can authenticate as.` aren't displaying correctly when viewing the file in GitHub.

Reply:
Good eye, thanks!